flash/text/classification/adapters.py

```diff
@@ -21,11 +21,11 @@
 from torch import Tensor
 
 from flash.core.adapter import Adapter, AdapterTask
+from flash.core.classification import CLASSIFIER_HEADS
```

Comment:
Nice!

Reply:
This as is like this because I added the `CLASSIFIER_HEADS` inside the `core/classification.py` should I move it to its own file `core/heads.py`

Reply:
Yeah, actually let's do it. Please move it to its own file: `core/heads.py`. Just to be consistent with other imports. :)